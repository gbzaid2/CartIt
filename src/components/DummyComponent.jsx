import React, { Component } from "react";

class DummyComponent extends Component {
  constructor() {
    super();
    this.state = {};
  }
  render() {
    return (
      <div className="container">
        <h1 className="title is-1">Hello World!</h1>
      </div>
    );
  }
}

export default DummyComponent;
