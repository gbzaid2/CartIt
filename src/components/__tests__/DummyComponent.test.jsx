import React from "react";
import { shallow } from "enzyme";
import DummyComponent from "../../components/DummyComponent.jsx";
import renderer from "react-test-renderer";

test("Header tag says 'Hello World'", () => {
  const wrapper = shallow(<DummyComponent />);
  const h1 = wrapper.find("h1");
  expect(h1.text()).toBe("Hello World!");
});

test("DummyComponent renders a snapshot properly", () => {
  const dummyComponent = renderer.create(<DummyComponent />).toJSON();
  expect(dummyComponent).toMatchSnapshot();
});
