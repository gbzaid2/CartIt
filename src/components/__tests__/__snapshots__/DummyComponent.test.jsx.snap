// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DummyComponent renders a snapshot properly 1`] = `
<div
  className="container"
>
  <h1
    className="title is-1 has-text-centered"
  >
    Hello World!
  </h1>
</div>
`;
